common/src/main/java/com/spredfast/kafka/connect/gcs/BytesRecordReader.java

```diff
@@ -30,6 +32,7 @@
  * Helper for reading raw length encoded records from a chunk file. Not thread safe.
  */
 public class BytesRecordReader implements RecordReader {
+	private static final Logger log = LoggerFactory.getLogger(BytesRecordReader.class);
 	private static final Gson GSON = new Gson();
 	private final ByteBuffer lenBuffer = ByteBuffer.allocate(LENGTH_FIELD_SIZE);
```

Comment:
Is this code going to be single-threaded? If multiple threads use the same instance of this class then there is certain to be data corruption.

Reply:
no it's a singleton

Reply:
I meant there should be only one instance / one thread

Reply:
There will be several tasks per pod - but in my understanding each of them would be a separate process so not multi-threading.

Reply:
Thanks for approval, asking a better Java dev than me (@ryanevansmettle) to check if he agrees with [this statement](https://github.com/eeveebank/kafka-connect-gcs/pull/14#discussion_r1097577888)

Reply:
I believe that to be correct.
Each task should create its own `BytesRecordReader` instance and not interfere with each other.